11-Syntax/03-Type-Level-Programming-Syntax/src/02-Basic-Syntax/04-Reflection.purs

```diff
@@ -31,61 +31,61 @@ instance tli_to_vlI :: TLI_to_VLI CustomKindInstance where {-
   reflectCustomKind _                       = "value-level instance"
 ----------------------------
 
--- An example using Booleans:
-data Boolean_ = True_Value | False_Value
+-- An example using the Boolean-like data YesNo:
+data YesNo = Yes | No
 
-foreign import kind BooleanKind
-foreign import data True :: BooleanKind
-foreign import data False :: BooleanKind
-data BooleanProxy (a :: BooleanKind) = BooleanProxyInstance
+foreign import kind YesNoKind
+foreign import data Yes :: YesNoKind
+foreign import data No  :: YesNoKind
+data YesNoProxy (a :: YesNoKind) = YesNoProxyInstance
```

Comment:
(I'm including this comment in case the question comes to mind due to other renaming comments in this review)

This could be renamed to `YesNoKindProxy`, but I think that's unnecessary. So, I think this should be left as you have it now.